### PR TITLE
Dynamic new page creation and add it to navigation menu automatically - Jira-116

### PR DESCRIPTION
**Pull request by rajguptaH**

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/BackOffice/UiPageTypeService.cs [hunk 1]
```diff
@@ -13,9 +13,11 @@ public class UiPageTypeService : IUiPageTypeService
     {
```
Comment (Sushilji) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/BackOffice/UiPageTypeService.cs:14]: Remove spacing

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/wwwroot/css/TestDetails.css [hunk 2]
```diff
@@ -314,6 +315,12 @@ form-group > label {
     border: 1px solid;
 }
 
+.dropdown-hover:hover .dropdown-menu {
```
Comment (Sumanthbhat04091999) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/wwwroot/css/TestDetails.css:318]: Don't write the CSS for the inbuilt class of bootsrap

If you required create a new class

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/wwwroot/css/TestDetails.css [hunk 3]
```diff
@@ -314,6 +315,12 @@ form-group > label {
...
+    display: block;
+}
+.dropdown-right {
```
Comment (Sumanthbhat04091999) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/wwwroot/css/TestDetails.css:321]: you need to change the class name bcs these are the inbuilt class name of the Boostrap
don't override

**Review by Sumanthbhat04091999 — COMMENTED** [review]
I have given the some comments in some of the Files.
In many files some unwanted line space is there, remove the unwanted line space.
In the CSS file for the inbuilt class of bootsrap CSS were written, it needed to be chnage

**Pull request merged**